src/content/structured/components/empty-state/code.mdx

```diff
@@ -27,27 +27,86 @@ export const snippets = [
   {
     language: "Web component",
     snippet: `<ic-empty-state 
-    heading="There's nothing here" 
-    subheading="We couldn't find anything to display here, but we’ll keep looking." 
-    body="Don’t worry, none of your content has been lost, we just can’t find what you’re looking for right now. Please try looking again.">
+    heading="Hmm...there's nothing here" 
+    subheading="We don't currently have any favourite teas, probably because coffee is much better." 
+    body="Take a look at our favourite coffees instead, there's much more of those.">
+    <svg slot="image" viewBox="0 0 1600 1600">
+      <rect fill="#ff7700" width="1600" height="1600" />
+      <polygon fill="#cc0000" points="957 450 539 900 1396 900" />
+      <polygon fill="#aa0000" points="957 450 872.9 900 1396 900" />
+      <polygon fill="#c50022" points="-60 900 398 662 816 900" />
+      <polygon fill="#a3001b" points="337 900 398 662 816 900" />
+      <polygon fill="#be0044" points="1203 546 1552 900 876 900" />
+      <polygon fill="#9c0036" points="1203 546 1552 900 1162 900" />
+      <polygon fill="#b80066" points="641 695 886 900 367 900" />
+      <polygon fill="#960052" points="587 900 641 695 886 900" />
+      <polygon fill="#b10088" points="1710 900 1401 632 1096 900" />
+      <polygon fill="#8f006d" points="1710 900 1401 632 1365 900" />
+      <polygon fill="#aa00aa" points="1210 900 971 687 725 900" />
+      <polygon fill="#880088" points="943 900 1210 900 971 687" />
+    </svg>
```

Comment:
Would these images be better being swapped for a coffee svg or something similar? They look a bit out of place now that all the other icons etc. have been swapped to coffee.

Reply:
@DBD324 Could not find suitable coffee SVG. Are you happy with the images currently?